ceno_zkvm/src/witness.rs

```diff
@@ -87,12 +87,12 @@ impl<T: Sized + Sync + Clone + Send + Copy> RowMajorMatrix<T> {
 
     pub fn par_batch_iter_padding_mut(
         &mut self,
-        num_rows: usize,
+        skip: usize,
```

Comment:
`skip` better be optional, and we can keep `let valid_instance = self.num_instances();` if it's None

Reply:
Done.